src/mods/land.js

```diff
@@ -130,7 +130,9 @@ idrinth.land = {
             document.getElementById ( 'idrinth-land-gold' ).value = idrinth.settings.get ( "land#gold" );
         };
         for (var key in idrinth.settings.get ( "land", true )) {
```

Comment:
would propose putting the resulting object into a variable out here somewhere, might be easier to read - otherwise fine with me :)